src/components/views/DownloadModModal.vue

```diff
@@ -57,7 +57,7 @@ import Profile from '../../model/Profile';
 import { Progress } from '../all';
 import ConflictManagementProvider from '../../providers/generic/installing/ConflictManagementProvider';
 import ModalCard from '../ModalCard.vue';
-import { installModsToProfile } from '../../utils/ProfileUtils';
+import {installModsAndResolveConflicts, installModsToProfile} from '../../utils/ProfileUtils';
```

Comment:
Nitpick: seems you've chosen to omit the spaces within braces here, although the convention in code base seems to favour them, at least in the context of imports.